sql/core/src/main/scala/org/apache/spark/sql/streaming/StreamingQueryInfo.scala

```diff
@@ -30,8 +30,15 @@ import org.apache.spark.annotation.Experimental
  * @param sinkStatus The current status of the [[StreamingQuery]]'s sink.
  */
 @Experimental
-class StreamingQueryInfo private[sql](
+case class StreamingQueryInfo private(
```

Comment:
I put it back because the toString of case class is incredibly helpful. How about I make the companion object private[sql] to avoid those being exposed to public?